content/desktop/release-notes.md

```diff
@@ -24,6 +24,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.25.1
+
+{{< release-date date="2023-11-13" >}}
+
+{{< desktop-install all=true version="4.25.1" build_path="/" >}}
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- The `sysctl vm.max_map_count` is now set to 262144. See [docker/for-mac#7047](https://github.com/docker/for-mac/issues/7047)
+
+#### For Mac
+
+- Fixed a bug when swap is disabled. Fixes [docker/for-mac#7045](https://github.com/docker/for-mac/issues/7045) and [docker/for-mac#7044](https://github.com/docker/for-mac/issues/7044).
```

Comment:
This is a fix for all platforms, so it needs to be moved in the "For all platforms" section. I also added a suggestion with the for-win issue.